primitives/src/traits/distribute_fees.rs

```diff
@@ -22,6 +22,8 @@ pub trait DistributeFees {
     type Balance;
     type MarketId;
 
+    fn get_fee(market_id: Self::MarketId, amount: Self::Balance) -> Self::Balance;
```

Comment:
I can do this, but why should we currently change it, if it is not required? I mean the API can adapt at any time if those parameters are needed.

Reply:
`distribute` allows the amount of distribution to depend on `asset` and `account`. The API should reflect that. Since you're introducing `get_fee` here, this is how I'd do it.